code/lib/codemod/src/transforms/__tests__/csf-2-to-3.test.ts

```diff
@@ -8,9 +11,17 @@ expect.addSnapshotSerializer({
 });
 
 const jsTransform = (source: string) =>
-  _transform({ source, path: 'Component.stories.js' }, null, {}).trim();
+  _transform(
+    { source, path: 'Component.stories.js' },
+    { jscodeshift: jscodeshift.withParser('tsx') } as API,
```

Comment:
Can you explain this `withParser('tsx')` bit? Is this because we're always defaulting to TSX now?